scripts/make-tarball.sh

```diff
@@ -17,5 +17,14 @@ git submodule foreach --quiet 'cd $toplevel && tar rf '"${TARBALL}"' $sm_path'
 ./scripts/autogen-version --store VERSION
 tar rf "${TARBALL}" VERSION
 
-# Compress archive.
-gzip -9 "${TARBALL}"
+# Introduce a top-level `spicy` directory and compress the tarball.
+SCRATCH=$(mktemp -d)
+mkdir "${SCRATCH}/spicy"
```

Comment:
Please include the version number into the directory name as well (as we do already for the binary tgzs).